app/scripts/modules/core/src/artifact/expectedArtifactMultiSelector.component.ts

```diff
@@ -33,34 +38,33 @@ const expectedArtifactMultiSelectorComponent: IComponentOptions = {
   controller: ExpectedArtifactMultiSelectorCtrl,
   controllerAs: 'ctrl',
   template: `
-      <render-if-feature feature="artifacts">
-        <ng-form name="artifacts">
-          <stage-config-field label="{{ctrl.artifactLabel}}" help-key="{{ctrl.helpFieldKey}}">
-            <ui-select multiple
-                       ng-model="ctrl.command[ctrl.idsField]"
-                       class="form-control input-sm expected-artifact-multi-selector">
-              <ui-select-match>
-                <img
-                  ng-if="ctrl.showIcons && ctrl.iconPath($item)"
-                  width="16"
-                  height="16"
-                  class="artifact-icon"
-                  ng-src="{{ ctrl.iconPath($item) }}" />
-                {{ $item.displayName }}
-              </ui-select-match>
-              <ui-select-choices repeat="expected.id as expected in ctrl.expectedArtifacts">
-                <img
-                  ng-if="ctrl.showIcons && ctrl.iconPath(expected)"
-                  width="16"
-                  height="16"
-                  class="artifact-icon"
-                  ng-src="{{ ctrl.iconPath(expected) }}" />
-                <span>{{ expected.displayName }}</span>
-              </ui-select-choices>
-            </ui-select>
-          </stage-config-field>
-        </ng-form>
-      </render-if-feature>
+      <ng-form name="artifacts" ng-if="ctrl.renderIf()">
```

Comment:
This is a big diff because of the re-indentation, but the only actual change is replacement of the `render-if-feature` wrapper with an `ng-if` conditional on `renderIf`.

Reply:
⚠️Just discovered that you can select ` Hide whitespace changes` from the ⚙️ dropdown; would recommend for this PR.